src/client/js/components/circle-chart.js

```diff
@@ -126,12 +126,17 @@ const styles = `
 </style>
 `
 
+/**
+ * @param {number} total
+ * @param {number} value
+ * @returns number
+ */
 const calcPercentage = (total, value) => {
   if (!total) {
     return 0
   }
 
-  return parseFloat((value / total).toFixed(3, 10))
+  return parseFloat((value / total).toFixed(3))
```

Comment:
`.toFixed` doesn't take a second argument.

Reply:
Q: does the `, 10)` belong to the `parseFloat()`?
I believe it defaults to base 10, but ESLint usually prefers an explicit value (as part of the [**radix**](https://eslint.org/docs/latest/rules/radix) rule, IIRC)?

```js
return parseFloat((value / total).toFixed(3), 10)
```

Reply:
Oh good one, yes, probably just a misplaced bracket, that makes more sense. Thanks: 2d806606a

Reply:
Oh actually, except that only [`parseInt` takes a `radix` parameter](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/parseInt), not [`parseFloat`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/parseFloat). Reverted in 1cc53d875.

Reply:
TIL! Good catch, sorry about that. Apparently I've been using `parseFloat()` wrong all this time (although I probably use `parseInt()` 99% of the time).